CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 All notable changes to this project will be documented in this file.
 
+## [4.0.4]
+
+### Added
+
+- Update to Wazuh version 4.0.4_1.11.0
```

Comment:
Hi @xr09,

Could link here the `wazuh/wazuh` `v4.0.4` changelog? https://github.com/wazuh/wazuh-ansible/blob/v4.0.4/CHANGELOG.md

Reply:
Fixed in 80bcd05